include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -3461,19 +3464,21 @@ FEEvaluationBase<dim, n_components_, Number, is_face, VectorizedArrayType>::
          ExcNotImplemented("Masking currently not implemented for "
                            "non-contiguous DoF storage"));
 
-  unsigned int n_vectorization_actual =
-    this->dof_info
-      ->n_vectorization_lanes_filled[this->dof_access_index][this->cell];
-
   const std::array<unsigned int, VectorizedArrayType::size()> &cells =
     this->get_cell_ids();
 
+  const unsigned int n_vectorization_actual =
+    std::count_if(cells.begin(), cells.end(), [](const auto i) {
+      return i != numbers::invalid_unsigned_int;
+    });
```

Comment:
You still have this code in - did you push an update yet?

Reply:
Now!